### PR TITLE
Add household major revision and fix several bugs

### PR DESCRIPTION
**Pull request by ecino**

### CODE REVIEW

File: child_compassion/models/icp_property.py [hunk 1]
```diff
@@ -128,6 +128,19 @@ def get_months_selection(self):
             ('Oct', _('October')),
             ('Nov', _('November')),
             ('Dec', _('December')),
+            # TODO See which format is sent
```
Comment (MickSandoz) [inline child_compassion/models/icp_property.py]: Do you want to keep this comment ?

File: child_compassion/models/field_office.py [hunk 2]
```diff
@@ -78,6 +78,32 @@ class FieldOffice(models.Model):
...
+        return field_office
+
+    ##########################################################################
+    #                             VIEW CALLBACKS                             #
+    ##########################################################################
+    @api.multi
+    def update_informations(self, context=None, async_mode=False):
+        """ Get the most recent informations for selected field offices and
+        update them accordingly. """
+        # TODO
```
Comment (MickSandoz) [inline child_compassion/models/field_office.py]: Do you want to keep this comment ?

Comment (ecino) [inline child_compassion/models/field_office.py]: Yes because it's still not done...

File: child_compassion/models/icp_property.py (hunk 1 above)
Comment (ecino) [inline child_compassion/models/icp_property.py]: Yes it's still not clear, (I asked U.S.)

**Comment by MickSandoz** [comment]
:+1:

**Pull request merged**